cmd/node-termination-handler.go

```diff
@@ -100,11 +101,23 @@ func main() {
 	for _, fn := range monitoringFns {
 		go func(monitor monitor.Monitor) {
 			log.Log().Msgf("Started monitoring for %s events", monitor.Kind())
+			var previousErr error
+			var duplicateErrCount int
 			for range time.Tick(time.Second * 2) {
 				err := monitor.Monitor()
 				if err != nil {
 					log.Log().Msgf("There was a problem monitoring for %s events: %v", monitor.Kind(), err)
 					metrics.ErrorEventsInc(monitor.Kind())
+					if err == previousErr {
+						duplicateErrCount++
+					} else {
+						duplicateErrCount = 0
+						previousErr = err
+					}
+					if duplicateErrCount >= duplicateErrThreshold {
+						log.Log().Msg("Stopping NITH - Duplicate Error Threshold hit.")
```

Comment:
Can you remove the "I" in NTH. 

"Stopping NTH - ..."

Reply:
Updated